src/python/pants/testutil/test_base.py

```diff
@@ -750,12 +750,13 @@ def get_bootstrap_options(self, cli_options=()):
     def make_snapshot(self, files):
```

Comment:
It'd be wonderful to add a type hint to this: `(files: Mapping[str, Union[str, bytes]]) -> Snapshot`

Reply:
Nice idea, done.